lib/cisco_node_utils/cmd_ref/bridge_domain.yaml

```diff
@@ -0,0 +1,44 @@
+# bridge_domain
+# bridge_domain feature is available only on n7k
+---
+_exclude: [N3k, N5k, N6k, N9k]
+
+all_bds:
+  multiple: true
+  get_command: "show bridge-domain"
+  get_value: '/^Bridge-domain (\d+) /'
+
+create:
+  set_value: "system bridge-domain add %s ; bridge-domain %s"
+
+destroy:
+  set_value: "no bridge-domain %s ; system bridge-domain remove %s"
+
+fabric_control:
+  get_command: "show bridge-domain summary"
+  get_value: '/^\w+ fabric-control [a-zA-Z0-9-_\/\s]+:\s+(\d+)/'
```

Comment:
Simplified the regex to /^[\S\s]+fabric-control[\S\s]+:\s+(\d+)/.
Also added the comment for the string to parse in yaml file.